R/subject_industry.R

```diff
@@ -823,16 +837,16 @@ crosstab_text <- function(tables_data_grouped, subjectinput, YAGinput, countinpu
 
```

Comment:
In line 809 above (or line 823 if going off the new line numbers), is there a stray whitespace before " the East of England"?